docs/rules/await-async-query.md

```diff
@@ -1,4 +1,4 @@
-# Enforce async queries to have proper `await` (await-async-query)
+# Enforce promise from async queries to be handled (await-async-query)
```

Comment:
Just updating the description and examples to be more accurate. We probably need to do the same on the others `await-async-*` rules.